packages/web-components/src/textarea/textarea.spec.ts

```diff
@@ -0,0 +1,915 @@
+import { test } from '@playwright/test';
+import type { Locator, Page } from '@playwright/test';
+import { expect, fixtureURL } from '../helpers.tests.js';
+import { TextAreaAppearance, TextAreaResize, TextAreaSize } from './textarea.options.js';
+import type { TextArea } from './textarea.js';
+
+test.describe('TextArea', () => {
+  let page: Page;
```

Comment:
This test format is older and runs all tests sequentially on the same page. so it's fragile in subtle ways and will likely break in the future (once Storybook gets upgraded). The file should be rewritten to follow the style set in `button.spec.ts` or `text-input.spec.ts`:
* Use one `test.beforeEach()` call to wait for the `fluent-textarea` custom element to be defined
* Utilize the `page` property from the first parameter, and scope locators and consts to each test
* Use `page.setContent()` instead of setting `root.innerHTML`

```ts
test.beforeEach(async ({ page }) => {
  await page.goto(fixtureURL('components-textarea--text-area'));

  await page.waitForFunction(() => customElements.whenDefined('fluent-textarea'));
});

test('example test', async ({ page }) => {
  const element = page.locator('fluent-textarea');
  const control = element.locator('textarea');

  await page.setContent(/* html */`
    <fluent-textarea required></fluent-textarea>
  `);

  await expect(control).toHaveJSProperty('required', true);
});
```

Reply:
This is great to know! I restructured the tests.